sklearn/naive_bayes.py

```diff
@@ -384,17 +421,11 @@ def _partial_fit(self, X, y, classes=None, _refit=False,
             # Put epsilon back in each time
             self.sigma_[:, :] -= epsilon
 
+        _check_y_classes(y, self.classes_)
```

Comment:
i see what you're saying. i'm computing it twice. i guess changing the helper to take unique values of y as input should work. though we may not always need unique values outside the helper, but we can always compute and pass it in.